config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java

```diff
@@ -1026,6 +1026,22 @@ public void default_distributor_three_phase_update_config_controlled_by_properti
         assertTrue(resolveThreePhaseUpdateConfigWithFeatureFlag(true));
     }
 
+    private int resolveMaxCompactBuffers(int maxCompactBuffers) {
+        VespaModel model = createEnd2EndOneNode(new TestProperties().maxCompactBuffers(maxCompactBuffers));
+        ContentCluster cc = model.getContentClusters().get("storage");
+        ProtonConfig.Builder protonBuilder = new ProtonConfig.Builder();
+        cc.getSearch().getConfig(protonBuilder);
+        ProtonConfig protonConfig = new ProtonConfig(protonBuilder);
+        assertEquals(1, protonConfig.documentdb().size());
+        return protonConfig.documentdb(0).allocation().max_compact_buffers();
+    }
+
+    @Test
+    public void default_max_compact_buffers_config_controlled_by_properties() {
+        assertEquals(2, resolveMaxCompactBuffers(2));
```

Comment:
Consider adding a test case for the default value of 1 (where maxCompactBuffers is not specified in TestProperties).

Reply:
Fixed.